cmd/container-disk-v2alpha/BUILD.bazel

```diff
@@ -0,0 +1,62 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+
+container_image(
+    name = "alpine-container-disk-image",
+    directory = "/disk",
+    files = ["@alpine_image//file"],
+    visibility = ["//visibility:public"],
+)
+
+container_image(
+    name = "cirros-container-disk-image",
+    directory = "/disk",
+    files = ["@cirros_image//file"],
+    visibility = ["//visibility:public"],
+)
+
+# used for e2e testing of custom base baths
+container_image(
+    name = "cirros-custom-container-disk-image",
+    directory = "/custom-disk",
+    files = ["@cirros_image//file"],
+    visibility = ["//visibility:public"],
+)
+
+container_image(
+    name = "fedora-cloud-container-disk-image",
+    directory = "/disk",
+    files = ["@fedora_image//file"],
+    visibility = ["//visibility:public"],
+)
+
+container_image(
+    name = "fedora30-cloud-container-disk-image",
+    directory = "/disk",
+    entrypoint = ["/entry-point.sh"],
```

Comment:
does this need the entry-point.sh still?

Reply:
Fixed